cypress/e2e/login.cy.js

```diff
@@ -1,11 +1,69 @@
 /// <reference types="cypress" />
 
 describe('Login page', () => {
+  const username = 'tomsmith';
+  const password = 'SuperSecretPassword!';
+
   beforeEach(() => {
-    
+    cy.visit('/login');
+  });
+
+  it('should provide an ability to log in with valid creds', () => {
+    cy.findById('username')
+      .type(username);
+
+    cy.findById('password')
+      .type(password + '{enter}');
+
+    cy.assertPageUrl('/secure')
+
+    cy.findById('flash')
+      .should('contain.text', 'You logged into a secure area!')
+      .should('be.visible')
```

Comment:
```suggestion
      .should('contain.text', 'You logged into a secure area!');
```
no need to add `be.visible`, previous assertion will do this job